Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -6,121 +6,212 @@ import Foundation
 class RouterSpec: QuickSpec {
     override func spec() {
         describe("A Router") {
-            var router = Router()
-
-            guard let urlOne = URL(string: Resource.test_get.rawValue),
-                let urlTwo = URL(string: Resource.test_head.rawValue),
-                let urlThree = URL(string: Resource.test_option.rawValue),
-                let urlNotFound = URL(string: "/not_here") else {
-                return
+            func buildRouter() -> Router {
+                return Router()
             }
-
-            context("when receive an invalid request object") {
-                let anInvalidRequest: HttpRequest? = nil
-                let expectedResponse = router.process(request: anInvalidRequest)
-                
+            
+            context("receives an invalid request object") {
                 it("returns a response with a 404 status code") {
-                    expect(expectedResponse.statusCode).to(equal(StatusCode.not_found.rawValue))
+                    let router = buildRouter()
+                    let anInvalidRequest: HttpRequest? = nil
+                    
+                    let response = router.process(request: anInvalidRequest)
+                    let expectedStatusCode = response.statusCode
+                    
+                    expect(expectedStatusCode).to(equal(StatusCode.not_found.rawValue))
```

Comment:
I would inline `expectedStatusCode` here.

Reply:
will fix that

Reply:
fixed